kube/services/jobs/fence-delete-expired-clients-cronjob.yaml

```diff
@@ -0,0 +1,114 @@
+---
```

Comment:
We don't need separate file for cronjob, this command  can use single definition and use it as  job or cronjob

```
gen3 job cron '* * * *' xxxx
```